src/components/ArticleList.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const ArticleList = () => {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        axios.get('https://sorei9240-nc-news.onrender.com/api/articles', {params: {limit: 100}})
```

Comment:
Just for clarification do you mean whole axios.get or just the limit per page? I had just put limit to 100 for now because I wanted to have all of the articles on a single page (my default is at 10) and was planing to incorporate the limit when I add in sorting and filtering